.env.example

```diff
@@ -3,7 +3,7 @@ URL=http://localhost:3000
 FIREBASE_ANALYTICS_ENABLED=0
 
 # Firebase configuration - https://firebase.google.com/docs/web/setup
-FIREBASE_CONFIG=
+FIREBASE_CONFIG={"apiKey":"AIzaSyAFrBPA02vd_aytIh4XHzqY7qEgCzp12qc","authDomain":"wedanceapp-d80f0.firebaseapp.com","projectId":"wedanceapp-d80f0","storageBucket":"wedanceapp-d80f0.appspot.com","messagingSenderId":"1025047528357","appId":"1:1025047528357:web:8080998384131b17d85e4c"}
```

Comment:
please don't change this file

Reply:
Yes, that's wedance's firebase info.
so we don't have to change that info.